java/src/main/kotlin/org/digma/intellij/plugin/idea/runcfg/constants.kt

```diff
@@ -6,3 +6,67 @@ const val OTEL_AGENT_JAR_URL = "https://github.com/open-telemetry/opentelemetry-
 const val DIGMA_AGENT_EXTENSION_JAR_URL = "https://github.com/digma-ai/otel-java-instrumentation/releases/latest/download/digma-otel-agent-extension.jar"
 const val OTEL_AGENT_JAR_NAME = "opentelemetry-javaagent.jar"
 const val DIGMA_AGENT_EXTENSION_JAR_NAME = "digma-otel-agent-extension.jar"
+
+val UNIMPORTANT_TASKS = listOf(
```

Comment:
I think need different lists for Maven and Gradle.
and maybe we can exclude groups , not sure if you seen via debug but for example test would be :test:abc , so there's this : thing